### ISSUE TITLE
Error when using example from readme

### ISSUE THREAD

**Issue by perstarkse** [Maroka-chan/VPN-Confinement#8]
Hi, 

Looking real interesting, thank you for sharing. 

When trying to use the example from the readme I get an error evaluating. 
``` error: attribute 'to' missing

       at /nix/store/1dsbdvxbjmm9abid199s01l4wkl8a5ad-source/modules/firewall-utils.nix:13:39:

           12|       "ip netns exec ${netns} iptables -A INPUT -p ${protocol}"
           13|       + " --dport ${builtins.toString x.to} -j ACCEPT -i ${interface}" + "\n")
             |                                       ^
           14|       (filter (m: !(builtins.isNull (builtins.match ("${protocol}|both") m.protocol))) portMappings);
```

Any ideas on what to try?

```
  vpnnamespaces.wg = {
    enable = true;
    wireguardConfigFile = config.sops.secrets."wg0.conf".path;
    accessibleFrom = [
      "192.168.0.0/24"
    ];
    portMappings = [
      { from = 9091; to = 9091; }
    ];
    openVPNPorts = [{
      port = 50909;
      protocol = "both";
    }];
  };

  # Add systemd service to VPN network namespace.
  systemd.services.transmission.vpnconfinement = {
    enable = true;
    vpnnamespace = "wg";
  };

  services.transmission = {
    enable = true;
    settings = {
      "rpc-bind-address" = "192.168.15.1"; # Bind RPC/WebUI to bridge address
    };
  };
```

**Comment by Maroka-chan** [comment]
whoops, that's my bad for not testing openVPNPorts after refactoring :x
Should be fixed by 907be6b09dfec9a805e0de915a745ec89e94ded8
Try updating :)

**Comment by perstarkse** [comment]
Thank you for the quick reply and fix, seems to have sorted it out :+1:
